### PR TITLE
Add rule for package pcsc-lite installed

### PR DESCRIPTION
**Pull request by yuumasato**
Select the rule in profiles that select service_pcscd_enabled.

#### Description:

- Add rule for package pcsc-lite installed
- Select the rule in profiles that select service_pcscd_enabled

#### Rationale:

- Rule `service_pcscd_enabled` fails because service doesn't exist when `pcsc-lite` is not installed.

#### Fixes:
```
TASK [Enable service pcscd] ***************************************************************************************************************************************************************************************
fatal: [rhel7.6]: FAILED! => {"changed": false, "msg": "Could not find the requested service pcscd: host"}
	to retry, use: --limit @/home/wsato/git/scap-security-guide/build/roles/ssg-rhel7-role-ospp.retry
```

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_pcsc-lite_installed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+rationale: |-
+    The pcsc-lite package must be installed if it is to be available for
+    multifactor authentication using smartcards.
+
+severity: medium
+
+references:
+    disa: "1954"
+    srg: SRG-OS-000375-GPOS-00160
+    vmmsrg: SRG-OS-000377-VMM-001530
```
Comment (yuumasato) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_pcsc-lite_installed/rule.yml]: Not sure if appropriate, but these are the same references as in `service_pcscd_enabled`

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_pcsc-lite_installed/rule.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8,fedora,rhv4
+
+title: 'Install pcsc-lite'
```
Comment (redhatrises) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_pcsc-lite_installed/rule.yml]: @yuumasato nitpick: `title: 'Install the pcsc-lite package'`

Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_pcsc-lite_installed/rule.yml]: Fixed.

**Comment by redhatrises** [comment]
LGTM

**Pull request merged**